cmd/tools/grafana/grafana.go

```diff
@@ -876,8 +883,16 @@ var Cmd = &cobra.Command{
 }
 
 var importCmd = &cobra.Command{
-	Use:     "import",
-	Short:   "import Grafana dashboards",
+	Use:   "import",
+	Short: "import Grafana dashboards",
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
maybe a comment along the line of

// Added so `directory` and `serverfolder` are required arguments except when both are empty. When both are empty use long accepted Harvest defaults